.zenodo.json

```diff
@@ -0,0 +1,39 @@
+{
+    "license": "cc-by-4.0",
+
+    "title": "Oxford Common File Layout - Extensions",
```

Comment:
Should the "Extensions" word be removed?

Reply:
Yes, fixed!